indica/converters/line_of_sight.py

```diff
@@ -75,7 +75,8 @@ def __init__(
     ):
 
         self.instrument_name: str = name
-        self.name = f"{name}_line_of_sight_transform"
+        self.type = "line_of_sight_transform"
```

Comment:
what do you need this for? the class name is self-explanatory and can be read within python if required

Reply:
I was imagining having some kind of dynamic behaviour based on the self.type, but that hasn't been useful yet.